test/exact_soln/vorticity_qoi.C

```diff
@@ -70,7 +71,9 @@ int main(int argc, char* argv[])
 
   // Initialize libMesh library.
   libMesh::LibMeshInit libmesh_init(argc, argv);
- 
+
+  GRINS::SimulationInitializer initializer;
+
   GRINS::SimulationBuilder sim_builder;
```

Comment:
Could we have SimulationBuilder create a SimulationInitializer?  Then we wouldn't need to change so much user code.